neuroinformatics/hbp_prov/hbp-prov-schema-v3.1.json

```diff
@@ -917,7 +913,7 @@
       "description": "Additional properties that can be attached to an entity. The properties have to be described in an ontology as features properties and hold information about the feature type and value restrictions if applicable.",
       "properties": {
         "key": {
-          "description": "The link to datatype property in the hbp attribute ontology",
+          "description": "The link to datatype property in the attribute ontology",
```

Comment:
I would say : 'An ontology term describing the attribute key type'

'The attribute ontology': 'The' attrbitute ontology is not described in this schema.